index.js

```diff
@@ -3032,6 +3032,22 @@
       [$.Array($.String), $.String],
       compose(R.join(''), R.map(concat(_, '\n'))));
 
+  //# append :: a -> [a] -> [a]
+  //.
+  //. Takes a value and a list of values of the same type and appends
+  //. the value to the list.
+  //.
+  //. ```javascript
+  //. > S.append(3, [1, 2])
+  //. [1, 2, 3]
+  //. ```
+  S.append =
+  def('append',
+      {},
+      [a, $.Array(a), $.Array(a)],
+      function(x, xs) { return xs.concat(x); });
```

Comment:
I think we need to wrap `x` here, currently we have `S.append([3, 4], [[1], [2]]); //=> [[1], [2], 3, 4]`